packages/rx_bloc_list/lib/src/models/identifiable_pair.dart

```diff
@@ -0,0 +1,8 @@
+import '../../models.dart';
+
+class IdentifiablePair<E extends Identifiable> {
+  IdentifiablePair({required this.old, required this.updated});
+
+  E old;
```

Comment:
Please document this class

Reply:
I have added comments on both of the fields